tests/runtest.sh

```diff
@@ -55,6 +55,10 @@ function print_usage {
     echo '                                     containing the coreclr source files'
     echo '  --coverage-output-dir=<path>     : Directory where coverage output will be written to'
     echo ''
+    echo 'Extra options:'
```

Comment:
Can you please move the options to the "Optional arguments" section and remove "Extra options"?